paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/PaymentSheetPlaygroundActivity.kt

```diff
@@ -214,11 +227,19 @@ class PaymentSheetPlaygroundActivity : AppCompatActivity() {
         }
 
         viewBinding.customCheckoutButton.setOnClickListener {
+            flowController.shippingDetails = shippingAddress
             flowController.confirm()
         }
 
+        viewBinding.shippingAddressButton.setOnClickListener {
+            startShippingAddressCollection()
+        }
+
+        viewBinding.shippingAddressContainer.visibility = View.GONE
```

Comment:
I think we should create a new activity to display the address element functionality, similar to the `PaymentSheet` and `PaymentSheet Custom` screens. This will make it easier to see the thing in action, instead of burying it in the messy playground.

Reply:
Can do that after this pull request is merged.